### PR TITLE
Adding build/release workflows

### PR DESCRIPTION
**Pull request by joseph-flinn**
## Overview
Adding in the build and release workflows to the browser extension. These are both pretty easy and straight forward.

## Code Changes
- **.github/workflows/build.yml:** creating the build workflow
- **.github/workflows/release.yml:** creating the release workflow

## Testing Considerations
There shouldn't be any.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.RELEASE_TAG_NAME }}
+          release_name: ${{ env.RELEASE_NAME }}
+          draft: true
+          prerelease: false
+
+
+  cli:
```
Comment (kspearrin) [inline .github/workflows/release.yml:81]: why is this called CLI?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      REPO_URL: ${{ needs.setup.outputs.repo_url }}
+      BUILD_NUMBER: ${{ needs.setup.outputs.adj_build_number }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup Windows builder
+        run: |
+          choco install checksum --no-progress
+          choco install reshack --no-progress
```
Comment (kspearrin) [inline .github/workflows/release.yml:94]: Shouldnt need this stuff. Was this copy/paste from somewhere else?

**Review by kspearrin — DISMISSED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (joseph-flinn) [reply .github/workflows/release.yml]: The pattern was copied over from cli and I forgot to rename it.

File: .github/workflows/release.yml (hunk 2 above)
Comment (joseph-flinn) [reply .github/workflows/release.yml]: yep, copied over from the cli

**Comment by joseph-flinn** [comment]
removed the testing code and the cloc job from release.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -70,8 +70,6 @@ jobs:
 
       - name: gulp
         run: gulp ci
-      - name: test dist
```
Comment (cscharf) [inline .github/workflows/build.yml]: Don't we need the test code on the build (just not the release)?

Comment (joseph-flinn) [reply .github/workflows/build.yml]: All that test code is doing is printing out all of the files in the `./dist` directory. I named that task poorly

**Review by cscharf — APPROVED** [review]

**Pull request merged**